setup.py

```diff
@@ -1,7 +1,6 @@
 import sys, glob, os, subprocess
 
-major, minor1, minor2, s, tmp = sys.version_info
-if major==2 and minor1<4 or major<2:
+if sys.version_info < (2, 4):
```

Comment:
I would be fine bumping the minimum version to at least 2.6 which is the oldest` one that we test with. Matplotlib has dropped support for 2.6 and earlier now